src/order/well_founded_set.lean

```diff
@@ -190,6 +190,11 @@ theorem partially_well_ordered_on.mono {s t : set α} {r : α → α → Prop}
   s.partially_well_ordered_on r :=
 λ f hf, ht f (set.subset.trans hf hsub)
 
+theorem is_pwo.mono [preorder α] {s t : set α}
```

Comment:
`anti`

Reply:
You're saying this should be called antimono? I just copied the lemma above for the name. I'll check against other examples of mono lemmas

Reply:
Yeah it's a small nit. Feel free to ignore me, I imagine I'm outnumbered by existing lemmas.

Reply:
I think this is my fault, as I named the other lemmas like that in this file? I forget if there is precedent of something like this elsewhere in the library...
In any case, I think the names like this in this file should be consistent - I'm inclined to merge this now, and then change all the names if there's a stronger consensus that this existing convention should be changed